lambdaStyle.css

```diff
@@ -0,0 +1,404 @@
+html, body {
+  margin: 0px;
+}
+button {
+  text-align: center;
+  background-color: rgb(66, 143, 187);
+  color: white;
+  display: inline-block;
+  margin-bottom: 0;
+  height: 4em;
+  width: 16em;
+  border-style: none;
+  text-decoration: none;
+}
+button, a {
+  color: white;
+  text-decoration: none;
+}
+button:hover {
+  cursor: pointer;
+}
+li {
+  display: inline-block;
+  margin: auto 1em;
+}
+li:hover {
+  cursor: pointer;
+}
+li ul {
+  display: none;
+  position: absolute;
+  background-color: rgb(35, 35, 35);
+  margin-top: 1.05em;
+}
+li ul li {
+  display: block;
+}
+li ul li p {
+  margin: 0;
+  font-size: 1em;
+}
+nav {
+  background-color: black;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 50px;
+}
+h1, h4 {
+  color: white;
+  text-align: center;
+  font-size: 4em;
+}
+h1 {
+  margin: 0 2em;
+  padding-top: 3em;
+}
+h2 {
+  color: black;
+  text-align: center;
+  margin-top: 0em;
+  margin-bottom: 2.5em;
+  font-size: 2em;
+  padding-top: 3em; 
+}
+h3 {
+  color: black;
+  margin: 7em 4em 0 4em;
+  font-size: 1.6em;
+}
+h4 {
+  font-size: 1.5em;
+  font-weight: normal;
+  margin: 1em 13em;
+  line-height: 1.75em;
+}
+h5 {
+  text-align: left;
+  float: left;
+  margin: 0;
+  display: block;
+  font-size: 1.1em
+}
+h6 {
+  text-align: center;
+  font-weight: normal;
+  margin: 4em;
+}
+hr {
+  width: 80%;
+}
+p {
+  color: rgb(165, 165, 165);
+  text-align: left;
+  font-size: 1.6em;
+  margin: 3em 10em;
+  line-height: 1.5em;
+}
+section {
+  display: inline-block;
+  width: 28%;
+  align-content: center;
+  text-align: left;
+}
+section p {
+  margin: 1em 0 5em 0;
+  float: left;
+  width: 95%
+}
+#blue {
```

Comment:
In general, if you're targeting this many id's, you should be able to generalize a lot of these to classes.